cosmic_ray/interceptors/spor.py

```diff
@@ -0,0 +1,32 @@
+"""An interceptor that uses spor metadata to determine when specific mutations
```

Comment:
GitHub won't allow me to leave a review comment on the empty `interceptors/__init__.py` file, so I'll leave one here.

I don't yet have a clear idea of what an "interceptor" is. I'd appreciate a brief explanation and a good place for this could be the module docstring for the `interceptors` package.